### PR TITLE
Rename database model class names

### PR DESCRIPTION
**Pull request by Mech0z**
## Description

This PR renames database classes to avoid having to specify full namespaces

## References

* #308

### CODE REVIEW

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/Context/Mapping/ChargeMapper.cs [hunk 1]
```diff
@@ -19,36 +19,34 @@
...
-using ChargeOperation = GreenEnergyHub.Charges.Infrastructure.Context.Model.ChargeOperation;
-using MarketParticipant = GreenEnergyHub.Charges.Infrastructure.Context.Model.MarketParticipant;
 
-namespace GreenEnergyHub.Charges.Infrastructure.Mapping
+namespace GreenEnergyHub.Charges.Infrastructure.Context.Mapping
 {
     public static class ChargeMapper
     {
-        public static Domain.Charge MapChargeToChargeDomainModel(Charge charge)
+        public static Domain.Charge MapChargeToChargeDomainModel(DBCharge dbCharge)
```
Comment (lasrinnil) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/Context/Mapping/ChargeMapper.cs:27]: Domain.Charge => Charge

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Infrastructure/Mapping/ChargeMapperTests.cs [hunk 2]
```diff
@@ -38,31 +36,31 @@ public void MapChargeToChargeDomainModel_SetsCorrectStartDateTimeFromDetails()
             var future = expected + TimeSpan.FromDays(3);
 
             // Act
-            var sut = ChargeMapper.MapChargeToChargeDomainModel(new Charge
+            var sut = ChargeMapper.MapChargeToChargeDomainModel(new DBCharge
```
Comment (lasrinnil) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Infrastructure/Mapping/ChargeMapperTests.cs:39]: Consider renaming the methods of the ChargeMapper to better reflect the new naming style.

MapDomainChargeToCharge -> MapDBChargeToCharge

**Review by lasrinnil — APPROVED** [review]

**Review by BjarkeMeier — COMMENTED** [review]
Doesn't these changes affect the database? I'd prefer if we can avoid renaming in the database - perhaps some EF Core convention based name handling? But at least you should make sure to not break the system.

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/Context/Mapping/ChargeMapper.cs (hunk 1 above)
Comment (Mech0z) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/Context/Mapping/ChargeMapper.cs]: Fixed in e94fa3ac30d2788b5c2475f408e29ac8bea13228

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Infrastructure/Mapping/ChargeMapperTests.cs (hunk 2 above)
Comment (Mech0z) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Infrastructure/Mapping/ChargeMapperTests.cs]: Changed in Fixed in e94fa3ac30d2788b5c2475f408e29ac8bea13228 and to fa45678c07f8230bb53aef94829c683c63280d4a
MapChargeContextModelToDomainModel and MapChargeDomainModelToContextModel

**Comment by BjarkeMeier** [comment]
It doesn't work and according to @Mech0z, the PR should be closed.

**Pull request closed without merging**